vector/src/main/java/im/vector/activity/LoginActivity.java

```diff
@@ -2009,6 +1999,67 @@ private void cancelEmailPolling() {
         }
     }
 
+    /**
+     * The user display name has been updated.
+     * @TODO remove this method as soon as the server is able to handle correctly the displayname
+     *
+     * @param logMessage the message to log
+     */
+    private void onDisplayNameUpdateDone(final String logMessage) {
+        Log.d(LOG_TAG, "## onDisplayNameUpdateDone " + logMessage);
+        finish();
+    }
+
+    /**
+     * Force the user display name
+     * @TODO remove this method as soon as the server is able to handle correctly the displayname
+     */
+    private void forceDisplayNameUpdate() {
+        final MXSession session = Matrix.getInstance(this).getDefaultSession();
+        session.getMyUser().refreshThirdPartyIdentifiers(new SimpleApiCallback<Void>() {
+            @Override
+            public void onSuccess(Void info) {
+                List<ThirdPartyIdentifier> currentEmail3PID = session.getMyUser().getlinkedEmails();
+
+                if (!currentEmail3PID.isEmpty()) {
+                    String emailAddress = currentEmail3PID.get(0).address;
+                    if (null != emailAddress) {
+                        String displayName = emailAddress.substring(0, emailAddress.lastIndexOf("@"));
+                        displayName = displayName.replace('.', ' ');
```

Comment:
The first letter of the first and last name could be capitalized.
String upperDisplayName = displayName.substring(0,1).toUpperCase() + displayName.substring(1);